operation.go

```diff
@@ -553,7 +565,7 @@ func setExample(param *spec.Parameter, schemaType string, value string) error {
 	if err != nil {
 		return nil // Don't set a example value if it's not valid
 	}
-	param.Example = val
+	param.Example = unescapeExample(val)
 	return nil
```

Comment:
Why are u making this change here?

Reply:
To also Support this the replacing in Example() , thats why i put it into a extra func

Reply:
any advice? remove or keep it for Example?

Reply:
Please remove the usage on Example. It may introduce a breaking change not covered by unit tests.

Reply:
done